explorer/src/entity/TaskRun.ts

```diff
@@ -29,9 +29,18 @@ export class TaskRun {
   @Column({ nullable: true })
   transactionStatus?: TransactionStatus
 
-  @Column({ nullable: true })
-  minimumConfirmations?: number
+  @Column({ nullable: true, name: 'minimumConfirmations_new1562419039813' })
+  minimumConfirmations?: string
 
-  @Column({ nullable: true })
-  confirmations?: number
+  @Column({ nullable: true, name: 'confirmations_new1562419039813' })
+  confirmations?: string
+
+  //
+  // TODO: Old columns will be deleted once node operators have migrated to this or
+  // later version.
+  @Column({ nullable: true, name: 'minimumConfirmations' })
```

Comment:
Could we explore something like this https://palantir.github.io/tslint/rules/deprecation/ just as a tool to catch people accidentally using this old field?

Reply:
I tried simply adding the 
```
/*
* @deprecated
*/
```
but that had no effect on our existing linter `yarn lint`. If you know of a way or a tool, please let me know.